### ISSUE TITLE
[Accessibility]: Popover element introduces tabindex

### ISSUE THREAD

**Issue by kopancek** [sourcegraph/sourcegraph#35713]
### Audit type

ARC Toolkit

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34113

### Problem description

When using the Popover element from wildcard library, it uses as a dependency `react-focus-lock`, which adds HTML elements with defined tabindex. These elements trigger errors when doing the ARC Toolkit check.

<img width="295" alt="Screenshot 2022-05-19 at 17 09 29" src="https://user-images.githubusercontent.com/9974711/169329825-70cbe6fb-5022-4758-bf45-c46d511a8240.png">

### Expected behavior

I would expect no errors from ARC toolkit

### Assigning labels

- [ ] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [ ] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

I'm unsure who should work on this issue or if it needs more clarification. I will let the Frontend Platform team triage this.
